UCSLibraries/src/main/java/it/cnr/iit/xacml/wrappers/PolicyWrapper.java

```diff
@@ -238,7 +257,7 @@ private RuleType getDefaultRuleType( String id, EffectType effectType ) {
      *          the condition to be put inside the new ruleType object
      * @return the ruleType object built in this way
      */
-    private RuleType copyRuleType( RuleType ruleType,
+    private RuleType cloneRuleType( RuleType ruleType,
             ConditionType conditionType ) {
```

Comment:
strange formatting. 261 can be on line 260

Reply:
yes! I hate these things :D